paasta_tools/kubernetes_tools.py

```diff
@@ -1599,6 +1603,22 @@ def create_pod_disruption_budget(
     )
 
 
+def set_instances_for_kubernetes_service(
+    kube_client: KubeClient, service_config: KubernetesDeploymentConfig, instance_count: int
+) -> None:
+    name = service_config.get_sanitised_deployment_name()
+    formatted_application = service_config.format_kubernetes_app()
+    formatted_application.spec.replicas = instance_count
+    if service_config.get_persistent_volumes():
+        kube_client.deployments.patch_namespaced_stateful_set_scale(name=name,
+                                                                    namespace="paasta",
+                                                                    body=formatted_application)
+    else:
+        kube_client.deployments.patch_namespaced_deployment_scale(name=name,
+                                                                  namespace="paasta",
+                                                                  body=formatted_application)
```

Comment:
Not a request for a fix, just a comment.

Was initially confused since it looks like this wouldn't play well with `setup_kubernetes_job`, since they would bump into each other. But @EvanKrall's PR #2709 fixes the issue using `get_autoscaled_instances` so it's fine.